doc/classes/Node3D.xml

```diff
@@ -224,143 +231,155 @@
+			[b]Note:[/b] Getting this property while the node is not inside the tree will fail and return [constant Transform3D.IDENTITY].
 		</member>
-		<member name="position" type="Vector3" setter="set_position" getter="get_position" default="Vector3(0, 0, 0)">
-			Local position or translation of this node relative to the parent. This is equivalent to [code]transform.origin[/code].
+		<member name="position" type="Vector3" setter="set_position" getter="get_position" default="Vector3(0, 0, 0)" keywords="translation">
+			Position (translation) of this node in local space (relative to the parent node). This is equivalent to the [member transform]'s [member Transform3D.origin].
 		</member>
 		<member name="quaternion" type="Quaternion" setter="set_quaternion" getter="get_quaternion">
-			Access to the node rotation as a [Quaternion]. This property is ideal for tweening complex rotations.
+			Rotation of this node represented as a [Quaternion] in local space (relative to the parent node). This value is obtained from [member basis]'s rotation.
+			[b]Note:[/b] Quaternions are much more suitable for 3D math but are less intuitive. Setting this property can be useful for interpolation (see [method Quaternion.slerp]).
 		</member>
 		<member name="rotation" type="Vector3" setter="set_rotation" getter="get_rotation" default="Vector3(0, 0, 0)">
-			Rotation part of the local transformation in radians, specified in terms of Euler angles. The angles construct a rotation in the order specified by the [member rotation_order] property.
-			[b]Note:[/b] In the mathematical sense, rotation is a matrix and not a vector. The three Euler angles, which are the three independent parameters of the Euler-angle parametrization of the rotation matrix, are stored in a [Vector3] data structure not because the rotation is a vector, but only because [Vector3] exists as a convenient data-structure to store 3 floating-point numbers. Therefore, applying affine operations on the rotation "vector" is not meaningful.
-			[b]Note:[/b] This property is edited in the inspector in degrees. If you want to use degrees in a script, use [member rotation_degrees].
+			Rotation of this node as [url=https://en.wikipedia.org/wiki/Euler_angles]Euler angles[/url], in radians and in local space (relative to the parent node). This value is obtained from [member basis]'s rotation.
+			- The [member Vector3.x] is the angle around the local X axis (pitch);
+			- The [member Vector3.y] is the angle around the local Y axis (yaw);
+			- The [member Vector3.z] is the angle around the local Z axis (roll).
+			The order of each consecutive rotation can be changed with [member rotation_order] (see [enum EulerOrder] constants). By default, the YXZ convention is used ([constant EULER_ORDER_YXZ]): Z (roll) is calculated first, then X (pitch), and lastly Y (yaw).
```

Comment:
Which is calculated first depends on whether you are composing or decomposing and whether you are considering intrinsic or extrinsic angles. I'm not sure if this note after the colon is helpful, it may be better to just have users click on `EULER_ORDER_YXZ` to read more there.